ISSUE_TRIAGE_PROCESS.rst

```diff
@@ -260,6 +260,21 @@ as long as the ``needs-triage`` label remains the triage team will keep an eye o
 if it needs to be accepted or closed/converted to Github Discussion.
 ``needs-triage`` label may also be applied manually by committers if they think a further action from the triage team is required.
 
+**area Label**
+
+``area:providers`` must be present for any provider issue. The ``provider:x`` is specifying the exact provider.
+We have ``provider:x`` for any provider that we own.
+``area:helm-chart`` must be for any helm chart issue.
+``area:core`` must be for any core issue. Additional labels like ``area:scheduler``, ``area:UI`` is specifying the exact
+core area relevant.
+
+This method allow us to quickly filter issues by the 3 major components of our code base: core, providers and helm-chart.
+
+**affected version Label**
+
+The ``affected_version:x`` is mandatory for core issues thus it always comes with ``area:core`` label.
```

Comment:
I am not sure if we can do it mandatory - especially that it will not be often present in the original issue. I guess this one shoudl be "best effort" - simply from practicality point of view.

Reply:
Generally speaking - I think we will never (including the dosu bot) get all of it 100% right, but we should be pretty realistic about quality of some of those labels IMHO.

Reply:
I am not sure if we can do it mandatory - especially that it will not be often present in the original issue. I guess this one shoudl be "best effort" - simply from practicality point of view.

The whole triage process is best effort, we work with what we have. Nonetheless this is what we should aim to have. This help us to see the most recent issues

Reply:
<img width="1161" alt="image" src="https://github.com/apache/airflow/assets/5144808/7584c12f-4dc1-4670-a46b-7a39661900de">

We probably could just use this version. At least this version is affected.

Reply:
> We probably could just use this version. At least this version is affected.

yes, normally this is what we do. sometimes we get more recent comment that the issue is reproducible on later versions so we modify label accordingly.